thoth/storages/graph/janusgraph.py

```diff
@@ -394,6 +394,8 @@ def get_solver_documents_count(self) -> dict:
             .has('solver_error') \
             .has('solver_error_unsolvable') \
             .has('solver_error_unparsable') \
+            .valueMap() \
+            .select('solver_document_id')
```

Comment:
before this to took all documents?

Reply:
I've realized it deduplicates based also on source/destination node - we simply counted all edges. We just want to count based on document ids.